wzdx/combination/attenuator.py

```diff
@@ -0,0 +1,190 @@
+import json
```

Comment:
We're missing quite a bit of unit test coverage in here, sitting at 30% currently. The only two functions covered are `combine_with_planned_events` and `validate_directionality`.

Reply:
And `get_distance_ahead`, missed that one.